test/e2e/infra/secondary_network/network-attachment-definition-crd.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
We mostly use ".yml" for YAML files. Better to change .yaml to .yml to be consistent.

Reply:
yes sure. taken care.